users/migrations/0021_remove_domain_ix_domain_state_next.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.2 on 2023-07-09 17:25
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("users", "0020_alter_identity_local"),
+    ]
+
+    operations = [
+        migrations.RemoveIndex(
+            model_name="domain",
+            name="ix_domain_state_next",
```

Comment:
This migration should not be here - please delete the migration file (the most recent `main` commit fixes the bug that makes this appear)

Reply:
Oups, forgot to leave it out the commit 😜 
Done